htlcswitch/hop/payload.go

```diff
@@ -258,15 +261,16 @@ func NewCustomRecords(parsedTypes tlv.TypeMap) record.CustomSet {
 // ensure that the proper fields are either included or omitted. The finalHop
 // boolean should be true if the payload was parsed for an exit hop. The
 // requirements for this method are described in BOLT 04.
-func ValidateParsedPayloadTypes(parsedTypes tlv.TypeMap,
-	isFinalHop bool) error {
+func ValidateParsedPayloadTypes(parsedTypes tlv.TypeMap, isFinalHop,
+	updateAddBlindingPoint, disableBlinding bool) error {
 
 	_, hasAmt := parsedTypes[record.AmtOnionType]
 	_, hasLockTime := parsedTypes[record.LockTimeOnionType]
 	_, hasNextHop := parsedTypes[record.NextHopOnionType]
 	_, hasMPP := parsedTypes[record.MPPOnionType]
 	_, hasAMP := parsedTypes[record.AMPOnionType]
 	_, hasEncryptedData := parsedTypes[record.EncryptedDataOnionType]
+	_, hasPayloadBlinding := parsedTypes[record.BlindingPointOnionType]
```

Comment:
Same q re just not even processing with decoding at all if we don't have blinded paths active.